src/game/client/c_world.cpp

```diff
@@ -15,6 +15,9 @@
 #include "eventlist.h"
 // NVNT haptic include for notification of world precache
 #include "haptics/haptic_utils.h"
+#if defined NEO && GLOWS_ENABLE
```

Comment:
Maybe should be `&& defined GLOWS_ENABLE` instead for consistency (since it's being ifdef'd elsewhere instead of checking for value).